### PR TITLE
[Workers] Limits page revision

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-8358

### CODE REVIEW

File: content/workers/platform/limits.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ title: Limits
 | [Worker size](#worker-size)                                                     | 1 MB      | 10 MB      |
 | [Worker startup time](#worker-startup-time)                                     | 200 ms    | 200 ms    |
 | [Number of Workers](#number-of-workers)                                         | 100       | 500       |
-| [Number of Cron Triggers<br/>per Worker](#number-of-schedules)                  | 3         | 3         |
-| [Number of Cron Triggers<br/>per account](#number-of-schedules-account)         | 5         | 250       |
+| Number of Cron Triggers<br/>per Worker                 | 3         | 3         |
+| Number of Cron Triggers<br/>per account         | 5         | 250       |
```
Comment (pedrosousa) [inline content/workers/platform/limits.md]:
```suggestion
| Number of Cron Triggers<br/>per Worker                                          | 3         | 3         |
| Number of Cron Triggers<br/>per account                                         | 5         | 250       |
```

File: content/workers/platform/limits.md [hunk 2]
```diff
@@ -126,24 +102,15 @@ Refer to [KV pricing](/workers/platform/pricing/#workers-kv) to review the speci
 
 {{</table-wrap>}}
 
-## Durable Objects limits
+- 50 total `put()`, `match()`, or `delete()` calls per-request, using the same quota as `fetch()`
 
-Durable Objects are only available on the Workers Paid plan.
+- 5 GBs total `put()` per request.
```
Comment (RebeccaTamachiro) [inline content/workers/platform/limits.md]:
```suggestion
- 5 GB total `put()` per request
```

File: content/workers/platform/limits.md [hunk 3]
```diff
@@ -265,6 +238,8 @@ App Workers do not count towards this limit.
 
 {{</Aside>}}
 
+---
+
 ## Number of routes per zone
 
 Each zone has a limit of 1,000 [routes](/workers/configuration/routing/routes/). If you require more than 1,000 routes on your zone, consider using [Workers for Platforms](/cloudflare-for-platforms/workers-for-platforms/) or request an increase to this limit by completing the [Limit Increase Request Form](https://forms.gle/ukpeZVLWLnKeixDu7).
```
Comment (RebeccaTamachiro) [inline content/workers/platform/limits.md:245]:
```suggestion
Each zone has a limit of 1,000 [routes](/workers/configuration/routing/routes/). If you require more than 1,000 routes on your zone, consider using [Workers for Platforms](/cloudflare-for-platforms/workers-for-platforms/) or request an increase to this limit by completing the [Limit Increase Request Form](https://forms.gle/ukpeZVLWLnKeixDu7).

---
```

**Review by RebeccaTamachiro — APPROVED** [review]

File: content/workers/platform/limits.md (hunk 1 above)
Comment (irvinebroque) [inline content/workers/platform/limits.md]:
```suggestion
|[Cron Triggers](/workers/configuration/cron-triggers/)<br/>per account         | 5         | 250       |
```

File: content/workers/platform/limits.md [hunk 4]
```diff
@@ -18,8 +18,8 @@ title: Limits
 | [Worker size](#worker-size)                                                     | 1 MB      | 10 MB      |
 | [Worker startup time](#worker-startup-time)                                     | 200 ms    | 200 ms    |
 | [Number of Workers](#number-of-workers)                                         | 100       | 500       |
-| [Number of Cron Triggers<br/>per Worker](#number-of-schedules)                  | 3         | 3         |
-| [Number of Cron Triggers<br/>per account](#number-of-schedules-account)         | 5         | 250       |
+| Number of Cron Triggers<br/>per Worker                 | 3         | 3         |
```
Comment (irvinebroque) [inline content/workers/platform/limits.md]:
```suggestion
|[Cron Triggers](/workers/configuration/cron-triggers/)<br/>per Worker                 | 3         | 3         |
```

File: content/workers/platform/limits.md [hunk 5]
```diff
@@ -75,45 +81,15 @@ Duration is a measurement of wall-clock time — the total amount of time from
...
-| Key metadata                          | 1024 bytes            | 1024 bytes  |
-| Value size                            | 25 MiB                | 25 MiB      |
-
-{{</table-wrap>}}
-
-{{<Aside type="note" header="Free versus Paid plan pricing">}}
-
-Refer to [KV pricing](/workers/platform/pricing/#workers-kv) to review the specific KV operations you are allowed under each plan with their pricing.
-
-{{</Aside>}}
```
Comment (irvinebroque) [inline content/workers/platform/limits.md:115]: I see, moved to https://developers.cloudflare.com/kv/platform/limits/, nice

**Review by irvinebroque — APPROVED** [review]
Can we find a way to clearly link to the new Durable Objects and KV docs pages for limits? So that someone who comes to this page, looking for info about DO or KV limits, can search within the page and find the right link?

Otherwise LGTM, thanks for the cleanup

**Pull request merged**